agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -343,6 +343,12 @@ private static void setOperationTags(AbstractTelemetryBuilder telemetryBuilder,
     setOperationName(telemetryBuilder, span.getAttributes());
   }
 
+  private static void setOperationTags(
+      AbstractTelemetryBuilder telemetryBuilder, String traceId, String parentSpanId) {
+    setOperationId(telemetryBuilder, traceId);
+    setOperationParentId(telemetryBuilder, parentSpanId);
+  }
+
```

Comment:
I think we don't need the changes in this file. If we don't, then good to revert them since it will likely cause conflicts with #2116